src/test/cypress/cypress/integration/Archives.spec.js

```diff
@@ -111,5 +121,8 @@ describe ('Archives screen tests',function () {
         // We check there is a plus icon in place of the minus icon
         cy.get('#opfab-archives-cards-list').find('.opfab-archives-table-line').first().find('.opfab-archives-icon-plus').should('have.length', 1);
         cy.get('#opfab-archives-cards-list').find('.opfab-archives-table-line').first().find('.opfab-archives-icon-minus').should('not.exist');
+
+        // We click collapsible updates
+        cy.get('#opfab-archives-collapsible-updates').click({force: true});
     })
```

Comment:
Add a  test without collapsible mode and check there is no + and the number of cards visible is correct

Reply:
Done.